velox/common/memory/ArbitrationParticipant.cpp

```diff
@@ -320,7 +320,7 @@ uint64_t ArbitrationParticipant::shrinkLocked(bool reclaimAll) {
 
 uint64_t ArbitrationParticipant::abort(
     const std::exception_ptr& error) noexcept {
-  ArbitrationOperationTimedLock l(reclaimMutex_);
+  std::lock_guard<std::timed_mutex> l(reclaimMutex_);
```

Comment:
Can we extend to take time for abort? Thanks!

Reply:
abort path is used by global arbitration. We currently don't have timeout information to pass for global arbitration because it does not belong to any requestor. The timeout protection is performed already when waiting for global arbitration.